cmd/asset-syncer/postgresql_utils.go

```diff
@@ -72,15 +71,13 @@ func (m *postgresAssetManager) Sync(repo models.Repo, charts []models.Chart) err
 	return m.removeMissingCharts(repo, charts)
 }
 
-func (m *postgresAssetManager) RepoAlreadyProcessed(repo models.Repo, repoChecksum string) bool {
+func (m *postgresAssetManager) LastChecksum(repo models.Repo) string {
 	var lastChecksum string
 	row := m.DB.QueryRow(fmt.Sprintf("SELECT checksum FROM %s WHERE name = $1 AND namespace = $2", dbutils.RepositoryTable), repo.Name, repo.Namespace)
 	if row != nil {
-		err := row.Scan(&lastChecksum)
-		log.Errorf("lastChecksum: %+v", lastChecksum)
-		return err == nil && lastChecksum == repoChecksum
+		row.Scan(&lastChecksum)
```

Comment:
There's an error return value here being ignored? Looks like the previous code returned false if there was an error, so you're updating that to match by returning "" if there was an error for the select. Is there a reason we don't just do `LastChecksum(repo models.Repo) (string, error)` to return and handle errors? Or does `row.Scan` return an error if the row doesn't exist - Ah ok, if that's the case, fine as is, though a comment to just short-circuit why we're not handling the error would be useful.

Reply:
[edit] At the end, it's not returning a typed error (`ErrNoRows`) as I thought but a different error when the entry doesn't exists: `sql: Scan error on column index 0, name "checksum": converting NULL to string is unsupported` which cannot be cleanly handled.